src/deltacode/__init__.py

```diff
@@ -58,6 +58,22 @@ def __init__(self, new_path, old_path, options):
         self.deltas = []
         self.errors = []
 
+        self._populate(new_path, old_path)
+        
+        self.new_scan_options = []
+        self.old_scan_options = []
```

Comment:
I forget where we stand on this, but if the VirtualCodebase objects contain this information, I do not see the value of our DeltaCode object containing it as well.

Reply:
VirtualCodebase is actually having all the resources but Delta objects will only contains only the respective deltas